### ISSUE TITLE
virtualbox: vboxdrv kernel module is not loaded

### ISSUE THREAD

**Issue by devopseze** [kubernetes/minikube#6812]
Creating virtualbox VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...

!   Unable to start VM: create: precreate: We support Virtualbox starting with version 5. Your VirtualBox install is "WARNING: The vboxdrv kernel module is not loaded. Either there is no module\n         available for the current kernel (4.15.0-1057-aws) or it failed to\n         load. Please recompile the kernel module and install it by\n\n           sudo /sbin/vboxconfig\n\n         You will not be able to start VMs until this problem is fixed.\n6.0.18r136238". Please upgrade at https://www.virtualbox.org

*   Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
-   https://github.com/kubernetes/minikube/issues/new

**Comment by RA489** [comment]
@devopseze  If you don't mind, could you please provide:
The exact command-lines used, so that we may replicate the issue
The full output of the command that failed
The full output of the "minikube logs" command
Which operating system version was used

**Comment by tstromberg** [comment]
@devopseze - What version of minikube was this with? Newer releases should give better guidance:

```
	"VBOX_DRV_NOT_LOADED": {
		Regexp: re(`vboxdrv kernel module is not loaded`),
		Advice: "Reinstall VirtualBox and reboot. Alternatively, try the kvm2 driver: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/",

		Issues: []int{4043, 4711},
	},
```

The underlying issue is the same though, this is indicative of a failed VirtualBox installation. Reinstall VirtualBox and it should be fine.

**Comment by devopseze** [comment]
Okay

**Comment by devopseze** [comment]
Am looking for documentation on how to install Minikube on Aws Ec2 Ubuntu instances all the ones I tried failed. Looking forward to hearing from you. 

Thanks 

Agbaeze

**Comment by tstromberg** [comment]
Please try `minikube start --driver=docker`

**Comment by tstromberg** [comment]
I'm closing this issue as it hasn't seen activity in awhile, and it's unclear if this issue still exists. If this issue does continue to exist in the most recent release of minikube, please feel free to re-open it by replying `/reopen`

If someone sees a similar issue to this one, please re-open it as replies to closed issues are unlikely to be noticed.

Thank you for opening the issue!

**Comment by medyagh** [comment]
I also have this problem on a google cloud instance
```
jenkins@ci-debian-agent1:~$ minikube start --driver=virtualbox
😄  minikube v1.11.0 on Debian 10.4
✨  Using the virtualbox driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🔥  Creating virtualbox VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
🤦  StartHost failed, but will try again: creating host: create: precreate: We support Virtualbox starting with version 5. Your VirtualBox install is "WARNING: The vboxdrv kernel module is not loaded. Either there is no module\n         available for the current ke
rnel (4.19.0-9-cloud-amd64) or it failed to\n         load. Please recompile the kernel module and install it by\n\n           sudo /sbin/vboxconfig\n\n         You will not be able to start VMs until this problem is fixed.\n6.1.10r138449". Please upgrade at https:
//www.virtualbox.org
🔥  Creating virtualbox VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
😿  Failed to start virtualbox VM. "minikube start" may fix it: creating host: create: precreate: We support Virtualbox starting with version 5. Your VirtualBox install is "WARNING: The vboxdrv kernel module is not loaded. Either there is no module\n         availa
ble for the current kernel (4.19.0-9-cloud-amd64) or it failed to\n         load. Please recompile the kernel module and install it by\n\n           sudo /sbin/vboxconfig\n\n         You will not be able to start VMs until this problem is fixed.\n6.1.10r138449". Pl
ease upgrade at https://www.virtualbox.org
❌  [VBOX_DRV_NOT_LOADED] error provisioning host Failed to start host: creating host: create: precreate: We support Virtualbox starting with version 5. Your VirtualBox install is "WARNING: The vboxdrv kernel module is not loaded. Either there is no module\n       
  available for the current kernel (4.19.0-9-cloud-amd64) or it failed to\n         load. Please recompile the kernel module and install it by\n\n           sudo /sbin/vboxconfig\n\n         You will not be able to start VMs until this problem is fixed.\n6.1.10r138
449". Please upgrade at https://www.virtualbox.org
💡  Suggestion: Reinstall VirtualBox and reboot. Alternatively, try the kvm2 driver: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/
⁉️   Related issues:
    ▪ https://github.com/kubernetes/minikube/issues/4043
    ▪ https://github.com/kubernetes/minikube/issues/4711
```
